mesop/components/input/input.py

```diff
@@ -40,6 +40,47 @@ class InputEnterEvent(MesopEvent):
 )
 
 
+@dataclass(kw_only=True, frozen=True)
+class Shortcut:
+  """Represents a keyboard shortcut combination."""
+
+  key: str
```

Comment:
I think it would be good to provide some documentation and (probably) validation on allowable keys. For example, "enter" is a common one. What about letters and numbers? It seems like we should only accept lowercase letters, e.g. "a", because "A" should really be "a" + shift=True

Reply:
Yeah that's a good point. Maybe we can just normalize input to lowercase. Since the key for enter is "Enter" so would be easier to just normalize that to "enter".

Looks like there's a table for all the potential key values here: https://developer.mozilla.org/en-US/docs/Web/API/UI_Events/Keyboard_event_key_values

I'll update the documentation to link to that.